docs/content/latest/develop/ecosystem-integrations/wso2.md

```diff
@@ -0,0 +1,179 @@
+---
+title: WSO2 Identity Server
+linkTitle: WSO2 Identity Server
+description: WSO2 Identity Server
+aliases:
+menu:
+  latest:
+    identifier: wso2
+    parent: ecosystem-integrations
+    weight: 571
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how to use [WSO2 Identity Server](https://wso2.com/identity-and-access-management/) to manage access to YugabyteDB.
+
+## Prerequisites
+
+Before you can start using WSO2 Identity Server, ensure that you have WSO2 Identity Server installed in one of the following location, depending on your operating system:
+
+- Mac OS:  /`Library`/`WSO2`/`IdentityServer`/*`<IS_VERSION>`*/*`<IS_HOME>`*
+- Windows:  `C:\Program Files\WSO2\IdentityServer`\\*`<IS_HOME>`*  
+- Ubuntu:  /`usr`/`lib`/`wso2`/`wso2is`/*`<IS_HOME>`*
+- CentOS:  /`usr`/`lib64`/`IdentityServer`/*`<IS_HOME>`*
+
+In addition, perform the following: 
+
+- Download and install [curl](https://curl.haxx.se/download.html).
+
+- Add the following entry to `/etc/hosts`:
+
+  ```
+  127.0.0.1    localhost.com
```

Comment:
Hey, sorry to be adding my two cents to this discussion completely out of the blue but I'd like to suggest not using a domain like this here. The problem with this is that your users will very often just copy this into their setups and this domain name is an actual registered domain belonging to someone: https://www.whois.com/whois/localhost.com. There were cases in the past where examples were using such example domains lead to information leak because the company releasing the examples either did not own the domain or did not extend the domain and someone else registered it. I believe a high profile case was Active Directory from Microsoft.

Consider using one of the reserved TLDs: https://tools.ietf.org/id/draft-chapin-rfc2606bis-00.html#new.

Reply:
Thanks for letting us know about this.

Reply:
@hbhanawat What should we use as "host" here?
CC: @m-iancu

Reply:
The steps have been borrowed from original WSO2 documentation. See here 
https://docs.wso2.com/display/IS570/Quick+Start+Guide
I would recommend we rather add a reference to WSO2 documentation rather than changing this.

Reply:
Added reference to the WSO2 doc.